#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raise a nameError exceptionwith a message"""
    raise NameError(message)
